osu.Game/Screens/Play/HUD/ArgonScoreCounter.cs

```diff
@@ -0,0 +1,124 @@
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Framework.Text;
+using osu.Game.Configuration;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public partial class ArgonScoreCounter : GameplayScoreCounter, ISerialisableDrawable
+    {
+        [SettingSource("Wireframe opacity", "Controls the opacity of the wire frames behind the digits.")]
+        public BindableFloat WireframeOpactiy { get; } = new BindableFloat(0.4f)
+        {
+            Precision = 0.01f,
+            MinValue = 0,
+            MaxValue = 1,
+        };
+
+        public bool UsesFixedAnchor { get; set; }
+
+        protected override IHasText CreateText() => new ArgonScoreTextComponent
+        {
+            WireframeOpactiy = { BindTarget = WireframeOpactiy },
+        };
+
+        private partial class ArgonScoreTextComponent : CompositeDrawable, IHasText
+        {
+            private readonly ArgonScoreSpriteText wireframesPart;
+            private readonly ArgonScoreSpriteText textPart;
+
+            public IBindable<float> WireframeOpactiy { get; } = new BindableFloat();
+
+            public LocalisableString Text
+            {
+                get => textPart.Text;
+                set
+                {
+                    wireframesPart.Text = value;
+                    textPart.Text = value;
+                }
+            }
+
+            public ArgonScoreTextComponent()
+            {
+                AutoSizeAxes = Axes.Both;
+
+                InternalChildren = new[]
+                {
+                    wireframesPart = new ArgonScoreSpriteText(@"wireframes"),
+                    textPart = new ArgonScoreSpriteText(),
+                };
+            }
+
+            protected override void LoadComplete()
+            {
+                base.LoadComplete();
+                WireframeOpactiy.BindValueChanged(v => wireframesPart.Alpha = v.NewValue, true);
+            }
+
+            private partial class ArgonScoreSpriteText : OsuSpriteText
+            {
+                private readonly string? glyphLookupOverride;
+
+                private GlyphStore glyphStore = null!;
+
+                protected override char FixedWidthReferenceCharacter => '5';
+
+                public ArgonScoreSpriteText(string? glyphLookupOverride = null)
+                {
+                    this.glyphLookupOverride = glyphLookupOverride;
+
+                    Shadow = false;
+                    UseFullGlyphHeight = false;
+                }
+
+                [BackgroundDependencyLoader]
+                private void load(ISkinSource skin)
+                {
+                    Font = new FontUsage(@"argon-score", 1, fixedWidth: true);
+                    Spacing = new Vector2(-2, 0);
```

Comment:
This could probably be avoided if the sprites didn't have huge whitespace padding, but I guess it's fine for now. We can readjust if/when this gets skin support.